torch/optim/rmsprop.py

```diff
@@ -4,21 +4,35 @@
 class RMSprop(Optimizer):
     """Implements RMSprop algorithm.
 
+    Proposed by G. Hinton in his `course <http://www.cs.toronto.edu/~tijmen/csc321/slides/lecture_slides_lec6.pdf>`_.
+
+    The centered version first appears in `Generating Sequences
+    With Recurrent Neural Networks <https://arxiv.org/pdf/1308.0850v5.pdf>`_.
+
     Arguments:
         params (iterable): iterable of parameters to optimize or dicts defining
             parameter groups
         lr (float, optional): learning rate (default: 1e-2)
+        momentum (float, optional): momentum factor (default: 0)
         alpha (float, optional): smoothing constant (default: 0.99)
         eps (float, optional): term added to the denominator to improve
             numerical stability (default: 1e-8)
+        centered (bool, optional) : if True, compute the centered RMSProp,
+            the gradient is normalized by an estimation of its variance
         weight_decay (float, optional): weight decay (L2 penalty) (default: 0)
 
     """
 
-    def __init__(self, params, lr=1e-2, alpha=0.99, eps=1e-8, weight_decay=0):
-        defaults = dict(lr=lr, alpha=alpha, eps=eps, weight_decay=weight_decay)
+    def __init__(self, params, lr=1e-2, momentum=0, alpha=0.99, eps=1e-8, centered=False, weight_decay=0):
```

Comment:
Ugh, sorry but I just realised that you inserted additional arguments at the beginning of `__init__`. We can't do that, because if someone created the optimizer like that:
```
optimizer.RMSprop(model.parameters(), 1e-2, 1 - 1e-4)
```
then the third argument will be interpreted as momentum, while it used to be `alpha`. Once that is change it's good to be merged.